.travis.yml

```diff
@@ -0,0 +1,27 @@
+before_script:
+  - ruby --version
+
+env:
+  global:
+    - APM_TEST_PACKAGES="language-haml linter"
```

Comment:
Can you try passing the package name as the parameter instead of letting it guess it? It should work that way. The magic `atom-package-deps` does to guess the name of the caller package doesn't with undefault or uncommon paths.

If that too doesn't work then I would recommend you load the package with `atom.packages.load` in a `beforeEach`. `atom-package-deps` gets the package manifest data using the Atom APIs. The Atom testing environment does not automatically load the package so you might have to do that manually.